intermine/imjs
# Implement loop query

## PR authors:
### Please describe your PR:

Fix for #48 - imjs was silently swallowing and killing loop path queries. it looks like it had never been implemented in the first place and the code simply didn't copy the loopPath attribute from one place to the next. 

To test:

The JSP interface for InterMine uses a CDN for many of the front end resources. To simulate this: 

1. Check this PR out.
2. Serve the dist directory (I did `serve -p 3333` with the npm serve package installed globally). 
3. In ~/.intermine/testmine.properties, add this line: `head.js.all.Ae = http://localhost:3333/im.js
` (change the port / path if needed). 

Then, to test everything in Testmine: 

4. in TestMine, run this query (you can import it in the QueryBuilder) :

```xml
<query model="testmodel" view="Department.manager.department.name Department.name" >
  <constraint path="Department.manager.department" op="=" code="A" loopPath="Department" />
</query>
```

If it's working, you should see something like this: 

<img width="722" alt="image" src="https://user-images.githubusercontent.com/9271438/71523928-17211800-28c3-11ea-9c39-3e5b33b2b6ae.png">
 
If this all works okay, let me know and I'll do the following: 

- [ ] Update [CHANGES](CHANGES) to reflect the new updates.
- [ ] Bump version in:
    - [ ] bower.json 
- [ ] release on npm
- [ ] add to CDN
